src/quantum/tests/GlobalVariable.test.ts

```diff
@@ -140,4 +194,29 @@ export class GlobalVariableTEst {
         }));
     }
 
+    public "Should not replace require statement 3"() {
+        return FuseTestEnv.create(
+            {
+                project: {
+                    files: {
+                        "index.ts": `
+                        const require = 2
+                        (function(){
+                            (function(){
+                                require('test')
+                            }())
+                        }())
+                    `
+                    },
+                    plugins: [QuantumPlugin({
+                        target: "browser"
+                    })]
+                }
+            }
+        ).simple().then(test => test.browser((window, env) => {
+            const app = env.getScript("app.js");
+            app.shouldNotFindString("require('test')");
```

Comment:
You are executing an invalid js here ;-) require is `2` should be a function instead

Reply:
![screen shot 2018-04-27 at 9 03 02 pm](https://user-images.githubusercontent.com/9585787/39366537-677cd536-4a5e-11e8-861f-4a7c441f6d9e.png)